airflow/www/views.py

```diff
@@ -108,8 +108,13 @@ def get_safe_url(url):
     valid_schemes = ['http', 'https', '']
     valid_netlocs = [request.host, '']
 
+    # Remove single quotes
+    url = url.replace("'", "")
     parsed = urlparse(url)
 
+    query = parse_qsl(parsed.query)
+    url = parsed._replace(query=urlencode(query)).geturl()
```

Comment:
Using a private method isn't great. Do we have to?

Reply:
It is NamedTuple's _replace method: https://docs.python.org/3/library/collections.html#collections.somenamedtuple._replace

Reply:
Right, so not private, just looks like it. Funsies.